### DISCUSSION TITLE
支持文章的多级标题

### DISCUSSION THREAD

**Discussion by chen-zhuohan** [drmingdrmer/md2zhihu#14] [Ideas]
目前知乎已经支持了给文章加目录，但是似乎markdown导入的文章，原本的多级标题都会被拍扁成一级标题。能否考虑支持一下markdown的多级标题 -> 知乎的多级标题？

**Comment by drmingdrmer** [comment]
测试了一下, 这应该是知乎markdown 导入程序的问题, markdown导入之后多级标题的信息丢失了.
